### PR TITLE
Add KeyringController tests

### PR DESCRIPTION
**Pull request by estebanmino**
This PR adds full coverage tests of `KeyringController`.

Resolves https://github.com/MetaMask/gaba/issues/66

### CODE REVIEW

File: src/KeyringController.ts [hunk 1]
```diff
@@ -184,7 +211,7 @@ export class KeyringController extends BaseController<BaseConfig, KeyringState>
 	 * @param password - Password of the keyring
 	 * @returns - Promise resolving to the seed phrase
 	 */
-	exportSeedPhrase(password: string): Promise<string> {
+	exportSeedPhrase(password: string) {
```
Comment (brunobar79) [inline src/KeyringController.ts]: Why did you remove the return type here?

Comment (estebanmino) [reply src/KeyringController.ts]: it doesn't return a `promise`, only a `string`
